cmd/dashboard.go

```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var dashboardCmd = &cobra.Command{
+	Use:   "dashboard",
+	Short: "Open dashboard on web UI",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// only support linux for now
+		return exec.Command("xdg-open", "https://paastech.cloud/#/dashboard").Start()
```

Comment:
maybe this?
[github.com/pkg/browser](https://github.com/pkg/browser)

Reply:
No need for another external dependency for such a trivial task.
xdg is enough for linux and windows and macos support will require more in-depth refactoring